.github/workflows/rust.yml

```
@@ -49,20 +54,31 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-directories: |
+            ~/.cargo/registry/src/**/librocksdb-sys-*
+
       - name: Lint with clippy
         uses: actions-rs/clippy-check@v1
```

actionlint

the runner of "actions-rs/clippy-check@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)